tutorial/westeros/westeros_firm_capacity.ipynb

```diff
@@ -40,29 +49,38 @@
    "outputs": [],
    "source": [
     "model = 'Westeros Electrified'\n",
-    "scen = base.clone(model, 'firm_capacity','illustration of firm-capacity formulation', keep_sol=False)\n",
-    "scen.check_out()\n",
-    "\n",
-    "year_df = scen.vintage_and_active_years()\n",
-    "vintage_years, act_years = year_df['year_vtg'], year_df['year_act']\n",
-    "model_horizon = scen.set('year')\n",
-    "country = 'Westeros'"
+    "base = message_ix.Scenario(mp, model=model, scenario='emission_bound')\n",
+    "scen = base.clone(model, 'firm_capacity','illustration of firm-capacity formulation', keep_solution=False)\n",
+    "scen.check_out()"
    ]
   },
   {
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "## add Renewbale Formulation\n"
+    "## Retrieve parameters to perform subsequent addition of parameters"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "year_df = scen.vintage_and_active_years()\n",
+    "vintage_years, act_years = year_df['year_vtg'], year_df['year_act']\n",
+    "model_horizon = scen.set('year')\n",
+    "country = 'Westeros'"
    ]
   },
   {
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "### Load Factors - Describing the Electricity Sector\n",
-    "#### Peak load \n",
-    "If we further specify our demand, we will find that the demand for electricity (!not the demand for useful light but for electricity!) has a peak load that is different from the annual average load; e.g: if annual average load is 5 GWh peak load during midday might be 10GW --> the peak load factor is equal to 2."
+    "## Load Factors - Describing the Electricity Sector\n",
+    "### Peak load \n",
+    "For systems across the U.S., load duration curves are similar from one location to another. Based on load data it was calculated calculated that the peak load is, on average, 1.7 times the average load (a load factor of 59%).8 Coupled with a standard reserve margin of 15–20% of peak load, that means firm capacity requirements for the U.S. should be roughly twice average load.\n",
+    "Therefore, a peak load factor of 2 is introduced for the level:secondary and commodity:electricity, which is level/commodity onto which all power generation technologies deliver."
```

Comment:
Please use some formatting for the level and commidity names. Also, "deliver" is jargon, maybe "where powerplant generation is accounted"?